api/src/Routes/testAPI.js

```diff
@@ -22,51 +17,25 @@ router.get("/test", function (req, res, next) {
   res.send({ text: "testAPI/test method" });
 });
 
-//bash script execute
-const X = "this is a param";
-
-// router.get("/bash", function (req, res, next) {
-//   exec("./bash.sh", (err, stdout, stderr) => {
-//     if (err) {
-//       console.error(err);
-//       return;
-//     }
-//     console.log(stdout);
-//   });
-
-//   res.send("Bash script executed here");
-// });
 
 router.post("/", urlencodedParser, function (req, res) {
-  console.log(req.body.url + " <- this is request body!!!!!!!");
-
-  const u = req.body.url;
-
-  res.send("response string from node");
-
-  // shell.echo(url);
-
-  // const ls = exec("ls", ["-lh", "/usr"]);
-
-  // ls.stdout.on("data", (data) => {
-  //   console.log(`stdout: ${data}`);
-  // });
-  // ls.on("close", (code) => {
-  //   console.log(`child process exited with code ${code}`);
-  // });
-
-  // shell.exec("cd ../");
-  // shell.exec("ls");
-
-  exec("bash.sh " + `'${u}'`, (err, stdout, stderr) => {
-    if (err) {
-      console.error(err);
-      return;
-    }
-    console.log(stdout);
-  });
-
-  //res.send("Bash script executed here");
+  const url = req.body.url;
```

Comment:
Please, change the route name as mentioned in the issue. 
Use try  and catch block to handle errors.